### PR TITLE
update language on app load, change initial value on timeperiod quest…

### PR DESCRIPTION
**Pull request by eilivjl**
…ions

use new dropdown in questionnaire, select correct initial value, scroll to top on iOS

### CODE REVIEW

File: src/components/questionnaire/InputNumberFormat.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ const setCursorPosition = (currentPos : any, priorText : string, currentText : s
     }
     return currentPos
 }
-/*function getOS() {
+function getOS() {
```
Comment (prabu-ssb) [inline src/components/questionnaire/InputNumberFormat.tsx:36]: kan du bruke `react-device-detect` i stedet ? se hvordan jeg plukker opp plattfom, nettleser info fra der, f.eks.

File: src/components/questionnaire/InputNumberFormat.tsx [hunk 2]
```diff
@@ -82,7 +82,7 @@ export default function InputNumberFormat({value, onChange = () => null, id, lab
         <div key={id} className={`ssb-input`}>
             {label && <label htmlFor={id}>{label}</label>}
             <div className="input-wrapper">
-            <input
+                {getOS() !== 'iOS' && <input
```
Comment (prabu-ssb) [inline src/components/questionnaire/InputNumberFormat.tsx:85]: it is precise these type of checks we should offload to react-device-detect

Comment (eilivjl) [reply src/components/questionnaire/InputNumberFormat.tsx:85]: kk

File: src/components/questionnaire/InputNumberFormat.tsx (hunk 1 above)
Comment (eilivjl) [reply src/components/questionnaire/InputNumberFormat.tsx:36]: det kan jeg sikkert, skal sjekke det ut

File: src/firebase/Neo.tsx [hunk 3]
```diff
@@ -360,6 +362,9 @@ const UserProvider = ({ children }) => {
         console.log('fetched preferences', prefs);
 
         const statuses = statusDoc.data() as UserStatusesType;
+        if(prefs && prefs.language) {
+          i18n.changeLanguage(prefs.language)
```
Comment (prabu-ssb) [inline src/firebase/Neo.tsx:366]: ah! this is what you were saying as missing the other day! :)

**Review by prabu-ssb — APPROVED** [review]
it looks good. i do not know the nitty-gritties of the questionnaire part and have not tested them, but they look good to me!

**Pull request closed without merging**